config/application.rb

```diff
@@ -78,46 +78,5 @@ class Application < Rails::Application
       config.action_mailer.default_url_options[:protocol] = "https"
     end
 
-    # Enable the asset pipeline
```

Comment:
Squash in to cb43aaee7511b95bbae97ebc26a83a5a7527750f

Mention http://railsdiff.org/4.0.13/4.1.16 so that we know where this change came from.